README.md

```diff
@@ -43,40 +43,110 @@ All these files generated are stored in inst/test files in your test package.
 RcppDeepState::deepstate_harness_compile_run("~/R/RcppDeepState/inst/testpkgs/testSAN")
 ```
 
-If all the function in the package are successfully compiled it gives the following message:
+It gives a list of functions that are successfully compiled in the package:
 
 ```R
```

Comment:
you should keep the README relatively short (only the most important info), so maybe move some of this to a vignette? https://r-pkgs.org/vignettes.html